pkg/environment/environment.go

```diff
@@ -389,14 +399,14 @@ func getEnvironmentFromParams(params kernel.Params) (Environment, error) {
 	// if the node running v4 chage urls to use v4 hub
 	if params.IsV4() {
 		env.FlistURL = defaultV4FlistURL
-		if flist := config.V4FlistURL; len(flist) > 0 {
-			env.FlistURL = flist
-		}
+		// if flist := config.V4FlistURL; len(flist) > 0 {
```

Comment:
this should come from zos-config, no?

Reply:
check https://github.com/threefoldtech/zosbase/pull/38#discussion_r2225633953